CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Features:
 
 Fix:
   - Schedule recipe selection
+  - Update `ActiveAdmin` to 1.1.0
+  - `FactoryGirl` update to `FactoryBot`
```

Comment:
The idea is to add all the changes to the *Unreleased* header until we bump the gem version and publish that new version. This way we can check in master if there any fixes or features that hasn't being released in a new version.

Now, it should look like this

```
## Unreleased

Fix: 
  - Update `ActiveAdmin` to 1.1.0
  - `FactoryGirl` update to `FactoryBot`
  
## 5.1.4
 
Changes:
  - Update ruby to 2.4
...
```